migrator/tests/actions/conftest.py

```diff
@@ -92,8 +98,8 @@ def test_extract_cls():
     class TestExtract(Extract):
         """Test extractor."""
 
-        def __init__(self, tx, filter_unchanged=True):
-            self.tx = tx
+        def __init__(self, txs, filter_unchanged=True):
+            self.txs = txs if isinstance(txs, list) else [txs]
```

Comment:
simply enable having more than one tx per file, but still back compat. it was needed for the gh disconnect, which is done in two phases.